data_discovery_ai/utils/es_connector.py

```diff
@@ -97,7 +97,7 @@ def search_es(
         current_pit = first_query_resp["pit_id"]
 
         # conduct further search
-        for r in tqdm(range(1, rounds), desc="searching elasticsearch"):
+        for r in range(1, rounds):
```

Comment:
It is ok for initial iteration, can we connect via OGC api later, I am worry about continue direct access to the json which make us hard to refactor the structure later, can you raise a ticket to explore this?